setup.py

```diff
@@ -78,7 +78,7 @@ def get_git_sha() -> str:
         "colorama",
         "croniter>=0.3.28",
         "cron-descriptor",
-        "cryptography>=39.0.1, <40",
+        "cryptography>=41.0.0",
```

Comment:
Can you put up a warning in the `UPDATING.md` file about this, since there are some backwards [incompatible changes](https://cryptography.io/en/latest/changelog/) from `39.x` to `41.x`?

Reply:
@betodealmeida I saw the Python 3.6 version which shouldn't affect Superset since we're already running on a minimum of 3.9, but do you think that the versions for `OpenSSL` and `LibreSSL` could be an issue? I'm wondering if this change would be considered a potential breaking change for Superset, and if so, if we need to think about cherrying it into 3.0 or wait until 4.0. I checked my local for example, and it seems likely that we could be running LibreSSL < 3.6

Reply:
cc @michael-s-molina or @rusackas